ipaserver/secrets/kem.py

```diff
@@ -181,7 +180,7 @@ def __init__(self, config=None, ipaconf=paths.IPA_DEFAULT_CONF):
         self.realm = conf.get('global', 'realm')
         self.ldap_uri = config.get('ldap_uri', None)
         if self.ldap_uri is None:
-            self.ldap_uri = conf.get('global', 'ldap_uri', None)
+            self.ldap_uri = conf.get('global', 'ldap_uri', raw=True)
```

Comment:
Or why not to use API when this is in ldap context? We have to find first why custodia related modules is not using api, ipalib